plugin/PDFContextMenu.qml

```diff
@@ -0,0 +1,135 @@
+/****************************************************************************************
+  **
+  ** Copyright (C) 2013-2016 Jolla Ltd., Damien Caliste
+  ** Contact: Martin Jones <martin.jones@jollamobile.com>
+  ** Contact: Damien Caliste <dcaliste@free.fr>
+  ** All rights reserved.
+  ** 
+  ** You may use this file under the terms of BSD license as follows:
```

Comment:
Yes, Silica is ok for source and even has the same license already. And you had even mentioned that earlier, sorry forgot that part.